Automattic/woocommerce-payments
# Update WCPay logo

Fixes #2345

#### Changes proposed in this Pull Request

This change updates the current WCPay logo with the new standardized logo available at PaqDfi-1me-p2 » Brand Guidelines PDF » Logo » Standard Lockup

SVG that is being proposed here is obtained from p1628527324102900/1628519670.089900-slack-C2CM2ELTG







#### Testing instructions

With a new installation of WC Pay,
* Navigate to Payments
* See that the WooCommerce Payments logo matches the logo available at PaqDfi-1me-p2 » Brand Guidelines PDF » Logo » Standard Lockup
* See that it responds to smaller device-width

After clicking "Finish Setup" and going through the setup process
* See that logo shown in Payments » Settings matches the new designs.

Note: "Recommended" pill overflow issue noted [here](https://github.com/Automattic/woocommerce-payments/pull/1600#pullrequestreview-639188238) still exists and not addressed with these changes.

Connection screen banner
![Connection screen banner](https://user-images.githubusercontent.com/6216000/132753301-fba18f05-13c4-4673-90c9-d5f2dc8bd930.png)

Settings page
![Settings page](https://user-images.githubusercontent.com/6216000/132753260-50721b05-b547-468d-8743-cd8124179021.png)






-------------------


**Post merge**
